src/vl53l5cx_class.h

```diff
@@ -148,7 +148,7 @@ class VL53L5CX {
      * @param (uint8_t) addr : New I2C address.
      * @return (uint8_t) status : 0 if init_sensor is OK.
      */
-    int init_sensor(uint8_t addr = VL53L5CX_DEFAULT_I2C_ADDRESS)
+    int init_sensor(uint16_t addr = VL53L5CX_DEFAULT_I2C_ADDRESS)
```

Comment:
I reject this fix because, by standard, I2C addresses are with 1 byte size, so it is correct to keep uint8_t size for the I2C address.